server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/datatransfer/model/DataTransferObject.java

```diff
@@ -13,35 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.oceanbase.odc.service.datatransfer.model;
 
-import com.oceanbase.tools.loaddump.common.enums.ObjectType;
+package com.oceanbase.odc.plugin.task.api.datatransfer.model;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
-import lombok.ToString;
 
-/**
- * odc export object which illustrate this map relationshit between name and db object
- *
- * @author yh263208
- * @date 2021-03-22 17:53
- * @since ODC_release_2.4.1
- */
-@Setter
-@Getter
-@ToString
+@Data
 @NoArgsConstructor
-@AllArgsConstructor
 public class DataTransferObject {
-    /**
-     * object type enum
-     */
-    private ObjectType dbObjectType;
```

Comment:
why should we use String instead of an enum?

Reply:
restored.